examples/timeline/other/drag&drop.html

```diff
@@ -27,6 +27,27 @@
       font-size: inherit;
       cursor: move;
     }
+    li.object-item {
+      list-style: none;
+      width: 150px;
+      color: #1A1A1A;
+      background-color: #D5DDF6;
+      border: 1px solid #97B0F8;
+      border-radius: 2px;
+      margin-bottom: 5px;
+      padding: 5px 12px;
+    }
+    li.object-item:before {
+      content: "≣";
```

Comment:
What is this for?

Reply:
so the example has a small symbol (which usually indicates it's draggable)